core-web/libs/sdk/editor/src/lib/models/editor.model.ts

```diff
@@ -0,0 +1,5 @@
+export enum NOTIFY_CUSTOMER {
+    EMA_RELOAD_PAGE = 'ema-reload-page', // We need to reload the ema page
+    EMA_REQUEST_BOUNDS = 'ema-request-bounds',
+    EMA_EDITOR_PONG = 'ema-editor-pong'
+}
```

Comment:
Add Docs and delete the one for `EMA_RELOAD_PAGE`. Also should we change the name since now it's for both VTL and EMA?

Reply:
Yes